src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container dialog-modal">
+    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content [formGroup]="dialogForm">
+        <mat-form-field>
+            <input matInput class="typedValueColor name-input" placeholder="Action Item Name" formControlName="name" />
+            <mat-error *ngIf="dialogForm.get('name').hasError('required')" class="invalid-feedback">
```

Comment:
`ngIf` shouldn't be used here. Please check documentation https://material.angular.io/components/form-field/overview#error-messages